include/ceed.h

```diff
@@ -289,9 +289,11 @@ CEED_EXTERN int CeedLobattoQuadrature(CeedInt Q, CeedScalar *qref1d,
 CEED_EXTERN int CeedQRFactorization(CeedScalar *mat, CeedScalar *tau, CeedInt m,
                                     CeedInt n);
 
+typedef int (*CeedQFunctionCallback)(void *, const CeedInt, const CeedScalar *const *, CeedScalar *const *);
```

Comment:
Perhaps CeedUserQFunction instead of CeedQFunctionCallback? Names are hard.

Reply:
Name modification done.